src/js/elements/tutorial.js

```diff
@@ -0,0 +1,17 @@
+let wrapperClass = '.w-slider-mask';
+let slideClass = '.w-slide';
+
+export class Tutorial {
+    createSlides = (tutorialArr) => {
+        if (tutorialArr === null || tutorialArr.length <= 0) {
```

Comment:
you could save a few lines and make the check implicit if you change this to:
```js
let tutorials = tutorialArr || [] 
```

Reply:
done